source/renderer/app/components/wallet/WalletSendForm.spec.js

```diff
@@ -0,0 +1,310 @@
+// @flow
+import React from 'react';
+import { IntlProvider, addLocaleData } from 'react-intl';
+import BigNumber from 'bignumber.js';
+import { Provider as MobxProvider } from 'mobx-react';
+import {
+  render,
+  fireEvent,
+  screen,
+  cleanup,
+  within,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import en from 'react-intl/locale-data/en';
+
+// Assets and helpers
+import translations from '../../i18n/locales/en-US.json';
+import StoryDecorator from '../../../../../storybook/stories/_support/StoryDecorator';
+import { sendFormAssetData } from '../../../../../storybook/stories/wallets/send/WalletSend.stories';
```

Comment:
IMO it would be good to make a copy of those guys in this file (or some some common test utils file if they are used by other tests too). This is because later while changing the storybook files someone might not be aware that those particular components are used elsewhere and changing them could brake the tests.